all-requirements.txt

```diff
@@ -1,14 +1,14 @@
-cliff>=2.0.0,<2.1.0
-eventlet!=0.18.3,!=0.20.1,<0.21.0,>=0.18.2
-gunicorn>=19.4.5
-lxml
+cliff>=2.11.0,<3.0.0
+eventlet>=0.22.0,<1.0.0
```

Comment:
Ubuntu Xenial only has 0.18.4, Ubuntu Bionic only has 0.20.0. Do we really need 0.22.0?